frontend/src/pages/RegisterPage.tsx

```
@@ -0,0 +1,124 @@
+          <Link to="/login" className="font-medium text-primary-600 hover:text-primary-500">
+            Sign in
+          </Link>
+        </p>
+      </div>
+
+      <div className="mt-8 sm:mx-auto sm:w-full sm:max-w-md">
+        <div className="card">
+          <form className="space-y-6">
+            {/* Role Selection */}
+            <div>
+              <label className="mb-3 block text-sm font-medium text-gray-700">I am a:</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)